pkg/virtual/apiexport/builder/build.go

```diff
@@ -52,7 +52,16 @@ import (
 	kcpinformers "github.com/kcp-dev/kcp/sdk/client/informers/externalversions"
 )
 
-const VirtualWorkspaceName string = "apiexport"
+const (
+	// VirtualWorkspaceName is the name of the virtual workspace.
+	VirtualWorkspaceName string = "apiexport"
+	// OriginalUserAnnotationKey is the key used in a user's "extra" to
+	// specify the original user of the authenticating request.
+	OriginalUserAnnotationKey = "authorization.kcp.io/original-username"
+	// OriginalGroupsAnnotationKey is the key used in a user's "extra" to
+	// specify the original groups of the authenticating request.
+	OriginalGroupsAnnotationKey = "authorization.kcp.io/original-groups"
```

Comment:
I would go with this as a stop-gap until the [warrant topic](https://github.com/kcp-dev/kcp/pull/3156) is resolved. Can you change these to `experimental.authorization.kcp.io/*`? Then I think we can merge.

Reply:
Done 👍